error: undefined reference to `__internal_intel_sub_group_2d_block_read_16b_16r8x4c_cache_controls`

When trying to add `inductor/test_max_autotune.py` to default Inductor test suite: https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/15633373081/job/44042609696

To reproduce: `python test/inductor/test_max_autotune.py TestPrologueFusion.test_upcast_sizes1`

Seems the same problem for:
* `python test/inductor/test_max_autotune.py TestPrologueFusion.test_upcast_sizes0`
* `python test/inductor/test_max_autotune.py TestPrologueFusion.test_multiple_inputs_sizes0`
* `python test/inductor/test_max_autotune.py TestPrologueFusion.test_multiple_inputs_sizes1`

Logs:
```bash

inductor/test_max_autotune.py::TestPrologueFusion::test_upcast_sizes0 L0 build module failed. Log: 
error: undefined reference to '__internal_intel_sub_group_2d_block_read_16b_16r8x4c_cache_controls'
in function: '__internal_intel_sub_group_2d_block_read_16b_16r8x4c_cache_controls' called by kernel: 'triton_tem_fused__to_copy_mm_0'

error: backend compiler failed build.

Error during Intel loadBinary: Triton Error [ZE]: 0x70000004

('RERUN', {'yellow': True}) [9.3997s] [ 87%]

```

cc @whitneywhtsang @alexbaden

--- a/test/TritonGEN/tritongen-2Dblockload-to-llvm.mlir
+++ b/test/TritonGEN/tritongen-2Dblockload-to-llvm.mlir
@@ -149,6 +149,20 @@ llvm.func @triton_gen.2Dblockload(%ptr : !llvm.ptr<1>, %base_width : i32, %base_
 // -----
 
 llvm.func @triton_gen.2Dblockload(%ptr : !llvm.ptr<1>, %base_width : i32, %base_height : i32, %base_pitch : i32, %x : i32, %y : i32) {
+  // CHECK:    %[[ELEM_BITS:.*]] = llvm.mlir.constant(16 : i32) : i32
+  // CHECK:    %[[TILE_WIDTH:.*]] = llvm.mlir.constant(8 : i32) : i32
+  // CHECK:    %[[TILE_HEIGHT:.*]] = llvm.mlir.constant(16 : i32) : i32
+  // CHECK:    %[[VBLOCKS:.*]] = llvm.mlir.constant(4 : i32) : i32
+  // CHECK:    %[[TRANSPOSE:.*]] = llvm.mlir.constant(false) : i1
+  // CHECK:    %[[VNNI:.*]] = llvm.mlir.constant(false) : i1
+  // CHECK:    %[[VAL_68:.*]] = llvm.call spir_funccc @llvm.genx.GenISA.LSC2DBlockRead.v32i16({{.*}}, {{.*}}, {{.*}}, {{.*}}, {{.*}}, {{.*}}, %[[ELEM_BITS]], %[[TILE_WIDTH]], %[[TILE_HEIGHT]], %[[VBLOCKS]], %[[TRANSPOSE]], %[[VNNI]], {{.*}})
+  %0 = triton_gen.2Dblockload %ptr, %base_width, %base_height, %base_pitch, %x, %y {elem_size_in_bits=16, tile_width=8, tile_height=16, v_blocks=4, transpose=false, vnni_transform=false, cache_control=Default} : (!llvm.ptr<1>, i32, i32, i32, i32, i32) -> vector<32xi16>
+  llvm.return
+}
+
+// -----
+
+llvm.func @triton_gen.2Dblockload(%ptr : !llvm.ptr<1>, %base_width : i32, %base_height : i32, %base_pitch : i32, %x : i32, %y : i32) {
   // CHECK:      llvm.mlir.constant(2 : i32) : i32
   // CHECK:      [[ElemSize:%.*]] = llvm.mlir.constant(2 : i32) : i32
   // CHECK-NEXT: [[TileWidth:%.*]] = llvm.mlir.constant(16 : i32) : i32

